ocean_lib/ocean/test/test_ocean_auth.py

```diff
@@ -31,9 +33,11 @@ def test_check_token(web3_instance):
     assert ocn_auth.check(sig) == "0x0"
 
     # Test token expiration
+    # TODO
```

Comment:
Was this meant to be completed in this ticket?

Reply:
no, I just replaced a FIXME with a TODO. I've also added this to the list of future improvements to the tests.